spring-xd-dirt/src/main/java/org/springframework/xd/dirt/server/admin/AdminServerApplication.java

```diff
@@ -189,4 +194,18 @@ public Filter httpPutFormContentFilter() {
 	public BatchDatabaseInitializer batchDatabaseInitializer() {
 		return new XdBatchDatabaseInitializer();
 	}
+
+
+	@Autowired
+	private ScheduledAnnotationBeanPostProcessor postProcessor;
+
+	@Autowired
+	@Qualifier("framework")
+	private TaskScheduler frameworkTaskScheduler;
+
+	@PostConstruct
+	public void forceScheduledTaskExecutor() {
+		postProcessor.setScheduler(frameworkTaskScheduler);
```

Comment:
+1 to Gary's comments, and...

I'm not a fan of the word "framework" here.

Also the thread prefix should be set.